plink-front/src/app/components/coin/coin.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="coin-container">
+  <p class="coin-name">{{coin.name}}</p>
+  <p class="coin-price">{{coin.price | currency}}</p>
+  <div class="coin-crypt">
+    <div *ngIf="crypto; else noCrypto" class="green-cont"><div class="led-green"></div>Si</div>
```

Comment:
I'd change this div to a span and change the inner div to use a ::before pseudoelement